Resources/public/css/theme/views/loginform.css

```diff
@@ -24,8 +24,9 @@
 }
 
 .ez-view-loginformview .ez-loginform-title {
-    font-size: 0.9rem;
+    font-size: 0.95rem;
     color: #444;
+    font-family: sans-serif;
```

Comment:
Hi! For me it is more a Marketing decision. Nothing beats the readability of Helvetica and its family, including Neue (just a few more maybe); That’s why I understand Damien’s question. But nothing like to visualize a transformation like a change of the primary font and due to we are precisely working for this… I would to try to be as consistent as possible throughout all spaces. Finally, Maven Pro compared to Helvetica Neue it is slightly thinner and curved, as well as increases the amount of space between letters. So maybe, in some cases, we should pay attention also to the font-weight or letter-spacing (specially for buttons, for instance) with Maven Pro.